packages/react-ui-components/src/stories/Components/TreeItem.stories.tsx

```diff
@@ -0,0 +1,126 @@
+import { EIconName, EOtherIconName } from '@kelvininc/ui-components';
+import { ComponentStory } from '@storybook/react';
+import React from 'react';
+import { KvTreeItem, ETreeItemStates } from '../../components';
+
+// Required to have the correct TagName in the code sample
+KvTreeItem.displayName = 'KvTreeItem';
+
+export default {
+	title: 'Navigation/Tree/Tree Item',
```

Comment:
Wouldn't this component be a better fit in the Data Display category? I understand the consideration regarding Navigation, but I'm unsure if it's the best category for it.